src/technologies.json

```
@@ -14376,6 +14376,18 @@
       "saas": true,
       "website": "https://newrelic.com"
     },
+    "Newspack": {
+      "cats": [
+        87
+      ],
+      "description": "Newspack is an open-source publishing platform built on WordPress for small to medium sized news organizations. It is an “opinionated” platform that stakes out clear, best-practice positions on technology, design, and business practice for news publishers.",
+      "dom": "link[href*='newspack.pub']",
```

Is this missing the `class`?

```suggestion
      "dom": "link.imprint[href*='newspack.pub']",
```

---

I think this should be `"dom": "a.imprint[href*='newspack.pub']"`.